CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 1.19.5-6
+
+* Set `client_body_buffer_size` to `128k`.
+* Set `client_header_timeout` to `605s`.
+* Set `client_max_body_size` to `500m`.
+* Set `keepalive_timeout` to `605s`.
+* Set `proxy_connect_timeout` to `60s` (same as default).
+* Set `proxy_send_timeout` to `60s` (same as default).
+* Set `send_timeout` to `60s` (same as default).
```

Comment:
I set these explicitly figuring it's better to know what these values are at a glance without having to reference the niginx docs.